lazyflow/operators/ioOperators/opTiffReader.py

```diff
@@ -125,11 +121,11 @@ def setupOutputs(self):
                     )
 
             if "q" in axes:
```

Comment:
Is this really possible?..

Reply:
yes, absolutely.

Reply:
What does it mean?.. I thought we only deal with "tzyxc".

Reply:
it means "unknown". Happens if you, e.g. add a signleton in `tifffile`...